sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -0,0 +1,366 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+// TODO: remove this
+#pragma warning(disable : 4239)
+
+#include "http/http.hpp"
+#include "common/xml_wrapper.hpp"
+
+#include <map>
+#include <string>
+#include <stdio.h>
+#include <time.h>
+#include <stdint.h>
+#include <ctime>
+
+namespace Azure
+{
+namespace Storage
+{
+namespace Blobs
+{
+class BlobRestClient
```

Comment:
Why do we need this `BlobRestClient` that includes everything? It looks like we  are adding another namespace for every class and functions.